2024-03-31 21:05:56 INFO [32m pytest tests/backend/test_create_project.py::TestProjectCreateWithInvalidData::test_create_project_with_invalid_ids[Roles.PROJECT_DEVELOPER-2979HjuWrZZ] (setup)[0m
curl -X GET 'http://localhost:8111/authenticationTest.html?csrf' \
-H 'User-Agent: python-requests/2.31.0'\
-H 'Accept-Encoding: gzip, deflate'\
-H 'Accept: */*'\
-H 'Connection: keep-alive'\
-H 'Authorization: Basic Ojg4MTE5MzQ2MjM3MjA4Mzg5NjA=' \
-d ''
2024-03-31 21:05:56 INFO [32m pytest tests/backend/test_create_project.py::TestProjectCreateWithInvalidData::test_create_project_with_invalid_ids[Roles.PROJECT_DEVELOPER-2979HjuWrZZ][0m
curl -X POST 'http://localhost:8111/app/rest/projects' \
-H 'User-Agent: python-requests/2.31.0'\
-H 'Accept-Encoding: gzip, deflate'\
-H 'Accept: application/json'\
-H 'Connection: keep-alive'\
-H 'Content-Type: application/json'\
-H 'X-TC-CSRF-Token: 624725ba-dc0a-4948-b0ce-5b28b98d4674'\
-H 'Cookie: RecentLogin=; TCSESSIONID=4EB46DFA0B698F86DFFE3FB239FBB50A'\
-H 'Content-Length: 114'\
-H 'Authorization: Basic Ojg4MTE5MzQ2MjM3MjA4Mzg5NjA=' \
-d '{"parentProject": {"locator": "_Root"}, "name": "project", "id": "2979HjuWrZZ", "copeAllAssociatedSettings": true}' 

2024-03-31 21:05:56 INFO 	RESPONSE:
STATUS_CODE: [31m500[0m
DATA: [31mError has occurred during request processing, status code: 500 (Internal Server Error).
Details: jetbrains.buildServer.serverSide.InvalidIdentifierException: Project ID "2979HjuWrZZ" is invalid: starts with non-letter character '2'. ID should start with a latin letter and contain only latin letters, digits and underscores (at most 225 characters).
Error occurred while processing this request.[0m